src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -5,14 +5,1235 @@
+    {
+        _settings = settings;
+    }
+
+    void TerminalPage::Create()
+    {
+        // Hookup the key bindings
+        _HookupKeyBindings(_settings->GetKeybindings());
+
+        _tabContent = this->TabContent();
+        _tabRow = this->TabRow();
+        _tabView = _tabRow.TabView();
+        _newTabButton = _tabRow.NewTabButton();
+
+        //Event Bindings (Early)
+        _newTabButton.Click([this](auto&&, auto&&) {
+            this->_OpenNewTab(std::nullopt);
+        });
+        _tabView.SelectionChanged({ this, &TerminalPage::_OnTabSelectionChanged });
+        _tabView.Items().VectorChanged({ this, &TerminalPage::_OnTabItemsChanged });
+        _tabView.TabClosing({ this, &TerminalPage::_OnTabClosing });
+
+        _CreateNewTabFlyout();
+        _OpenNewTab(std::nullopt);
+
+        _tabContent.SizeChanged({ this, &TerminalPage::_OnContentSizeChanged });
+    }
+
+    // Method Description:
+    // - Register our event handlers with the given keybindings object. This
+    //   should be done regardless of what the events are actually bound to -
+    //   this simply ensures the AppKeyBindings object will call us correctly
+    //   for each event.
+    // Arguments:
+    // - bindings: A AppKeyBindings object to wire up with our event handlers
+    void TerminalPage::_HookupKeyBindings(TerminalApp::AppKeyBindings bindings) noexcept
+    {
+        // Hook up the KeyBinding object's events to our handlers.
+        // They should all be hooked up here, regardless of whether or not
+        // there's an actual keychord for them.
+        bindings.NewTab([this]() { _OpenNewTab(std::nullopt); });
+        bindings.DuplicateTab([this]() { _DuplicateTabViewItem(); });
+        bindings.CloseTab([this]() { _CloseFocusedTab(); });
+        bindings.ClosePane([this]() { _CloseFocusedPane(); });
+        bindings.NewTabWithProfile([this](const auto index) { _OpenNewTab({ index }); });
+        bindings.ScrollUp([this]() { _Scroll(-1); });
+        bindings.ScrollDown([this]() { _Scroll(1); });
+        bindings.NextTab([this]() { _SelectNextTab(true); });
+        bindings.PrevTab([this]() { _SelectNextTab(false); });
+        bindings.SplitVertical([this]() { _SplitVertical(std::nullopt); });
+        bindings.SplitHorizontal([this]() { _SplitHorizontal(std::nullopt); });
+        bindings.ScrollUpPage([this]() { _ScrollPage(-1); });
+        bindings.ScrollDownPage([this]() { _ScrollPage(1); });
+        bindings.SwitchToTab([this](const auto index) { _SelectTab({ index }); });
+        bindings.OpenSettings([this]() { _OpenSettings(); });
+        bindings.ResizePane([this](const auto direction) { _ResizePane(direction); });
+        bindings.MoveFocus([this](const auto direction) { _MoveFocus(direction); });
+        bindings.CopyText([this](const auto trimWhitespace) { _CopyText(trimWhitespace); });
+        bindings.PasteText([this]() { _PasteText(); });
+    }
+
+    void TerminalPage::RefreshUIAfterSettingsReloaded()
```

Comment:
I'd maybe also call it `RefreshUIForSettingsReload`, simply because the `UIA` in the method name the way it is makes my brain think it's related to UIA, even though it _really_ isn't.

Reply:
changed name and added comment